### PR TITLE
Fix tour of beam co-group-by-key python example

### LINKED ISSUE #29535: [Bug]: CoGroupByKey Tour of Beam Python example shows Go code
### What happened?

I am going through the "Tour of Beam", and the CoGroupByKey Python example shows Go code instead of Python.  It can be seen in the playground starting a bit below here:

https://github.com/apache/beam/blob/3e0d0d62a5b1a9063f9e07a62556ba8a7299afbe/learning/tour-of-beam/learning-content/core-transforms/map/co-group-by-key/description.md#playground-exercise

I am happy to make a PR and fix it, but being new to Beam, I wasn't sure if that would help.

### Issue Priority

Priority: 3 (minor)

### Issue Components

- [X] Component: Python SDK
- [ ] Component: Java SDK
- [ ] Component: Go SDK
- [ ] Component: Typescript SDK
- [ ] Component: IO connector
- [ ] Component: Beam YAML
- [X] Component: Beam examples
- [X] Component: Beam playground
- [ ] Component: Beam katas
- [X] Component: Website
- [ ] Component: Spark Runner
- [ ] Component: Flink Runner
- [ ] Component: Samza Runner
- [ ] Component: Twister2 Runner
- [ ] Component: Hazelcast Jet Runner
- [ ] Component: Google Cloud Dataflow Runner

### PR DESCRIPTION
**Pull request by damccorm**
Right now, this renders go code for the python example, this fixes the problem.

Fixes #29535

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

**Comment by damccorm** [comment]
R: @riteshghorse

**Review by riteshghorse — APPROVED** [review]
LGTM, thank you!

File: learning/tour-of-beam/learning-content/core-transforms/map/co-group-by-key/description.md [hunk 1]
```diff
@@ -284,42 +284,28 @@ static PCollection<String> applyTransform(PCollection<String> fruits, PCollectio
...
-	weightIter(&wa.ProductWeight)
-	fruitIter(&wa.Fruit)
-    emit(wa.String())
-
-	}, grouped)
-}
+def apply_transforms(fruits, weights):
+    def cogbk_result_to_product_weight(cgbk_result):
+        (country, values) = cgbk_result
+        return WordsAlphabet(values['weights'][0], values['fruits'][0], country)
```
Comment (riteshghorse) [inline learning/tour-of-beam/learning-content/core-transforms/map/co-group-by-key/description.md:305]:
```suggestion
        return ProductWeight(values['weights'][0], values['fruits'][0], country)
```

**Comment by riteshghorse** [comment]
Sorry I missed the `ProductWeight`, upon checking the whole file, WordsAlphabet is not defined for python

**Comment by riteshghorse** [comment]
I can send a PR. The java code also has some corrections

**Comment by damccorm** [comment]
Ah good catch

> I can send a PR. The java code also has some corrections

Sounds good, thanks

**Pull request merged**